.github/workflows/deploy-application.yml

```diff
@@ -149,10 +149,91 @@ jobs:
     environment: ${{ inputs.environment }}
     steps:
       - run: echo "Proceeding with deployment to $environment"
+
+  run-migrations:
+    name: Run migrations
+    runs-on: ubuntu-latest
+    needs: [ approve-deployments ]
+    if: ${{ inputs.server_types == 'all' || inputs.server_types == 'ops' }}
```

Comment:
Not sure we want to only run this if we're deploying "all" or the "ops" server. Currently the migrations run no matter what, and if we want to control whether it runs or not maybe a checkbox, or something, would be better.